test/parallel/test-process-execve-permission-fail.js

```diff
@@ -0,0 +1,21 @@
+// Flags: --permission --allow-fs-read=*
+
+'use strict';
+
+const { mustCall, skip, isWindows } = require('../common');
+const { fail, throws } = require('assert');
+const { isMainThread } = require('worker_threads');
+
+if (!isMainThread) {
+  skip('process.execve is not available in Workers');
+} else if (isWindows) {
+  skip('process.execve is not available in Windows');
+}
+
+if (process.argv[2] === 'replaced') {
+  fail('process.execve should not have been allowed.');
+} else {
+  throws(mustCall(() => {
+    process.execve(process.execPath, [process.execPath, __filename, 'replaced'], process.env);
+  }), { name: 'Error', code: 'ERR_ACCESS_DENIED' });
```

Comment:
We should have a `permission: 'ChildProcess'` check here. No need to check `name`.

Reply:
Will do very soon. Thank!

Reply:
Added, thanks!